package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wasm4s",
+  "name": "scala-wasm",
```

Comment:
Oops, my local repository was names `wasm4s` for no reason, and I just renamed it to `scala-wasm` that matches with the remote repo 😅